DependencyInjection/HttplugExtension.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Http\ClientBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+/**
+ * @author David Buchmann <mail@davidbu.ch>
+ */
+class HttplugExtension extends Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $configuration = $this->getConfiguration($configs, $container);
+        $config = $this->processConfiguration($configuration, $configs);
+
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+
+        $loader->load('discovery.xml');
+        foreach ($config['classes'] as $service => $class) {
+            if ($class) {
```

Comment:
What do you check here? Maybe a more explicit check (isset, empty, value check)?